trellis-app/src/main/java/org/trellisldp/app/TrellisApplication.java

```diff
@@ -56,6 +64,28 @@
  */
 public class TrellisApplication extends Application<TrellisConfiguration> {
 
+    private Environment environment;
+
+    private TrellisConfiguration config;
+
+    private EventService notificationService;
+
+    private IdentifierService idService;
+
+    private MementoService mementoService;
+
+    private TriplestoreResourceService resourceService;
```

Comment:
Presumably, a subclass would not use this field? In other words, doesn't this add a hard dependency on the Triplestore implementation?